data/Abhijeet-Gautam5702.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Abhijeet Gautam",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student. Passionate frontend developer, aspiring to go fullstack. Love problem solving.",
+  "avatar": "https://github.com/Abhijeet-Gautam5702.png",
```

Comment:
```suggestion
  
```
remove the avatar line it will automatically fetch your real time github profile pic

Reply:
Alright, @baibhav-tiwari, I'll do that. Thanks.